Vineyard 2023 Annual Review

sighingnow opened the pull request:
Following the guidance: https://github.com/cncf/toc/blob/main/process/sandbox-annual-review.md#how-to-file-your-annual-review

Resolves https://github.com/v6d-io/v6d/issues/1561

mattfarina reviewed (approved):
Thank you for filing the annual review. I have the following feedback for you:

- Devstats states that most contributions come from _Independent_. It appears that Alibaba is the organization that should be linked for many developers. Can the developers please update their affiliations in https://github.com/cncf/gitdm.
- Can you please update the CNCF logo on the project website to use a sandbox version. See https://github.com/cncf/artwork/blob/main/examples/other.md#cncf-sandbox-logos. I added https://github.com/v6d-io/website/issues/3 for this task.
- There is a repository for the v6d.io website but it has no source for the website. Can the source for the website please be open up. I added https://github.com/v6d-io/website/issues/2 to support this.
- The "New Committers in this year" section is exactly the same as the 2022 annual report. If there were no new committers it could have been removed or noted. If there was a different list of new committers that list should have been used instead.
- I like that you noted _PingAn_ moving from testing to production usage. Looking across years I can see the transition.

As the project is under active development while being used and tested by adopters this PR is being approved.

sighingnow commented:
Hi @mattfarina, thanks a lot for the feedback!

> * Devstats states that most contributions come from _Independent_. It appears that Alibaba is the organization that should be linked for many developers. Can the developers please update their affiliations in https://github.com/cncf/gitdm.

I have tried my best to update the affiliation of contributors from Alibaba Group in gitdm and submitted a pull request (https://github.com/cncf/gitdm/pull/240) for more accurate statistics.

> * Can you please update the CNCF logo on the project website to use a sandbox version. See https://github.com/cncf/artwork/blob/main/examples/other.md#cncf-sandbox-logos. I added [Update v6d.io use of CNCF logo v6d-io/website#3](https://github.com/v6d-io/website/issues/3) for this task.

Fixed in https://github.com/v6d-io/v6d/pull/1636.

> * There is a repository for the v6d.io website but it has no source for the website. Can the source for the website please be open up. I added [v6d.io site source v6d-io/website#2](https://github.com/v6d-io/website/issues/2) to support this.

The website was once created for collaboration and the source is finally available in the v6d-io/v6d repo. I have archived the website repo.

> * The "New Committers in this year" section is exactly the same as the 2022 annual report. If there were no new committers it could have been removed or noted. If there was a different list of new committers that list should have been used instead.

It was indeed a mistake. I have submitted a pull request for a quick fix: https://github.com/cncf/toc/pull/1221

> * I like that you noted _PingAn_ moving from testing to production usage. Looking across years I can see the transition.

Thank you!

> As the project is under active development while being used and tested by adopters this PR is being approved.

It's really awesome to be part of CNCF communicty!
